content/news/2021-XX-XX-fsf-statement.md

```diff
@@ -0,0 +1,25 @@
+title: "A Statement on Richard Stallman and the Free Software Foundation"
+authors: Owen Williams
+tags: community, diversity, politics
+date: 2021-03-26 00:00:00
+
+The contributors to the Mixxx project were dismayed this week to learn that Richard Stallman had been reinstated to the board of directors of the Free Software Foundation.
+His departure in 2019 was a consequence of the criticism he faced regarding his inappropriate behavior towards women, his hurtful comments about people with Down's Syndrome and his remarks relating to the Epstein scandal.
+His resignation came as a relief for many who have suffered his bad behavior.
+Richard Stallman's actions disqualify him from representing the FSF and the free software community as a whole.
+
+While Mixxx is not officially associated with the Free Software Foundation, there's an inevitable connection in the public's mind between all FOSS projects and the FSF.
+Furthermore, Mixxx is and always has been licensed under the terms of the GNU General Public License, version 2 or *later*, and only the Free Software Foundation can publish new versions of the GNU GPL which could be retroactively applied to the Mixxx code.
+Given this relationship, the Mixxx project has an interest in ensuring that the Free Software Foundation has inclusive, principled, and wise leadership which can effectively lead the broad free software community to meet present and future challenges.
+
+Therefore, we feel an obligation to add our voices to the chorus of those who have expressed dismay at this turn of events and call for the resignation of Richard Stallman.
+The FSF board of directors, by allowing him to return, is giving the appearance of endorsing his conduct and can't remain effective without significant personnel changes and bylaw reforms.
+By sanctioning his return, the board of directors has lost all credibility on the issues of diversity, equity, and inclusion.
+We also call upon the Free Software Foundation to revise its bylaws to institute democratic processes for choosing leadership and increase the transparency of its operations.
+The myriad software developers who chose to trust the FSF with stewardship of licenses for the code we wrote deserve to have a voice on the leadership of the FSF.
+We cannot trust an organization that knowingly makes controversial decisions behind closed doors.
+
+The FOSS community has led the way for the rest of the wider tech industry, showing that open standards and practices result in better products while respecting users' freedom, privacy, and community.
+Now we have the opportunity to light a better way for the industry at large by showing that a more welcoming, inclusive industry also creates products that are better for everyone.
+Our community needs to demonstrate a strong sense of accountability at this time.
+We urge the FSF board to do the right thing by accepting responsibility for this bad decision and enact specific commitments for change to rebuild trust.
```

Comment:
```suggestion
We now expect the FSF board to take responsibility for this bad decision and making enact specific commitments for change to rebuild trust.
```
This way it matches to the current development. 

> We urge the

We can urge them in a letter addressed to them and we can tell our users about it.

Reply:
I think "urge" fits better to be honest. And I think we can use it here, as it's basically kind of an open letter.

Reply:
Deepl translates this to: 

Wir fordern den FSF-Vorstand auf, das Richtige zu tun, indem er die Verantwortung für diese Fehlentscheidung übernimmt und konkrete Änderungszusagen macht, um das Vertrauen wiederherzustellen.

But we don't, we write a news blog. 

We could write: We have asked the FSF board ...
If we had done that, but so far we have not. 

Therefore I think it is correct to express our expectations here. In the sense of a last chance. 
Otherwise, we must clearly distance ourselves from the FSF, as Red Head has done. 
A contribution in this sense I would like more, but so it is also OK.

Reply:
Even in German, "urge" ("auffordern") would be correct IMHO. We do it by publishing this article (like an open letter). "We expect" ("wir erwarten") sounds very antagonistic to me. But I don't have a hard opinion to this and leave it to @ywwg to decide.

Reply:
My idea was to acknowledge that they already have announced this step.

Reply:
Ok, maybe something like:
```suggestion
We expect the FSF board to do the right thing and to follow up its statements with actions by accepting responsibility for this bad decision and enact specific commitments for change to rebuild trust.
```

Reply:
Thank you. @ywwg The "expect" was meant like the angry mother expect when telling her son to tidy up. And not like expecting a snail post letter. Don't know if both versions exists in English.

Reply:
That's good, just change "enact" to "enacting".

Reply:
Done.